### PR TITLE
CPP-426 - Proxy Authorization and Execution

### PR DESCRIPTION
**Pull request by stamhankar999**
* (WIP) Added set_execute_as method to Statement.

### CODE REVIEW

File: cpp-driver/src/statement.hpp [hunk 1]
```diff
@@ -109,7 +109,6 @@ class Statement : public RoutableRequest, public AbstractData {
   std::string paging_state_;
   uint8_t kind_;
   std::vector<size_t> key_indices_;
-
```
Comment (mpenick) [inline cpp-driver/src/statement.hpp]: Why u no like this newline? :)

File: include/dse.h [hunk 2]
```diff
@@ -217,6 +217,48 @@ cass_cluster_set_dse_gssapi_authenticator_n(CassCluster* cluster,
...
+ * @public @memberof CassCluster
+ *
+ * @param[in] cluster
+ * @param[in] service
+ * @param[in] principal
+ * @param[in] authorization_id
+ * @return CASS_OK if successful, otherwise an error occurred.
+ */
+DSE_EXPORT CassError
+cass_cluster_set_dse_gssapi_authenticator_with_authorization(CassCluster* cluster,
```
Comment (mpenick) [inline include/dse.h]: `cass_cluster_set_dse_gssapi_authenticator_proxy` shorter, potentially more descriptive?

File: cpp-driver/src/statement.hpp (hunk 1 above)
Comment (stamhankar999) [inline cpp-driver/src/statement.hpp]: That was an accident; I think I had originally added some methods, then removed them in a later iteration, and the blank line got lost along the way. Will fix.

File: include/dse.h (hunk 2 above)
Comment (stamhankar999) [inline include/dse.h]: I like it. I was struggling with this a bit but couldn't come up with something better at the time and figured you'd come up with something. :)

**Pull request merged**